### ISSUE TITLE
Random user picture w/o API

### ISSUE THREAD

**Issue by cfjedimaster** [RandomAPI/Randomuser.me-Data#41]
Maybe consider the ability to point an image tag to the site (randomuser.me/randimg) that would spit out a random user picture. So I could simply drop a few img tags in and get random user pictures.

**Comment by keitharm** [comment]
That sounds like a cool idea. It could also accept a gender parameter as well.

**Comment by cfjedimaster** [comment]
And size too. I think size would be required.

**Comment by keitharm** [comment]
Yes, that'd be a nice parameter to have. Also I could see `seed` working here as well.

Would this just output an actual image, or would it send the URLs back in JSON format?
If the latter, a `results` parameter could also be provided that sends back an array of URLs.

**Comment by cfjedimaster** [comment]
No JSON at all. I literally want to do this:

<img src="http://randomuser.me/img">

Other examples:

(makes a 40x40)
<img src="http://randomuser.me/img?s=40">

(makes a woman)
<img src="http://randomuser.me/img?g=f">

(makes a man, 80x80)
<img src="http://randomuser.me/img?g=m&s=80">



On Wed, Jul 15, 2015 at 8:10 AM, Keith Armstrong <notifications@github.com>
wrote:

> Yes, that'd be a nice parameter to have. Also I could see seed working
> here as well.
>
> Would this just output an actual image, or would it send the URLs back in
> JSON format?
> If the latter, a results parameter could also be provided that sends back
> an array of URLs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RandomAPI/Randomuser.me-Data/issues/41#issuecomment-121610055>
> .
>



-- 
===========================================================================
Raymond Camden, Developer Advocate for MobileFirst at IBM

Email : raymondcamden@gmail.com
Blog : www.raymondcamden.com
Twitter: raymondcamden

**Comment by jessmartin** [comment]
:+1:
